<!DOCTYPE html>
<html lang="en-us">
  <head>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    
    <title>Chapter 3 Preclinical Studies to Enable First in Human Trials - Illiterate&#39;s blog</title>
    <meta property="og:title" content="Chapter 3 Preclinical Studies to Enable First in Human Trials - Illiterate&#39;s blog">
    
    <meta name="twitter:card" content="summary">

    
      
    

    
      
      <meta property="description" content="Preclinical genomic data
- https://depmap.org/portal/
- https://www.sanger.ac.uk/
- https://cansarblack.icr.ac.uk/
[&amp;hellip;] Target selection
[&amp;hellip;] Target validation based on
[&amp;hellip;] ​ - &amp;hellip;">
      <meta property="og:description" content="Preclinical genomic data
- https://depmap.org/portal/
- https://www.sanger.ac.uk/
- https://cansarblack.icr.ac.uk/
[&amp;hellip;] Target selection
[&amp;hellip;] Target validation based on
[&amp;hellip;] ​ - &amp;hellip;">
      
    

    
    
    <meta name="twitter:image" content="https://i.loli.net/2021/03/08/HiM91TKd7EyulGg.png">
    
    

    

    
    

    <link rel="stylesheet" href="/css/style.css" />
    <link rel="stylesheet" href="/css/fonts.css" />
    
  </head>

  
  <body class="posts">
    <header class="masthead">
      <h1><a href="/">Illiterate&#39;s blog</a></h1>

<p class="tagline">Personal website of illiterate</p>

      <nav class="menu">
  <input id="menu-check" type="checkbox" hidden/>
  <label id="menu-label" for="menu-check" class="unselectable" hidden>
    <span class="icon close-icon">✕</span>
    <span class="icon open-icon">☰</span>
    <span class="text">Menu</span>
  </label>
  <ul>
  
  
  <li><a href="/">首页</a></li>
  
  <li><a href="/posts">日志</a></li>
  
  <li><a href="/categories/">分类</a></li>
  
  <li><a href="/tags/">标签</a></li>
  
  <li><a href="about/">关于</a></li>
  
  
  </ul>
</nav>

    </header>

    <article class="main">
      <header class="title">
      
<h1>Chapter 3 Preclinical Studies to Enable First in Human Trials</h1>

<h3>
  2021-03-08</h3>
<hr>


      </header>



<p>Preclinical genomic data</p>
<p>- <a href="https://depmap.org/portal/">https://depmap.org/portal/</a></p>
<p>- <a href="https://www.sanger.ac.uk/">https://www.sanger.ac.uk/</a></p>
<p>- <a href="https://cansarblack.icr.ac.uk/">https://cansarblack.icr.ac.uk/</a></p>
<h3 id="key-steps-in-pre-clinical-drug-discovery-before-fih">Key steps in pre-clinical drug discovery before FiH</h3>
<ul>
<li>
<p>Target selection</p>
</li>
<li>
<p>Target validation based on</p>
</li>
</ul>
<p>​     - Biological assessment</p>
<p>​     - Technical assessment</p>
<ul>
<li>
<p>Strategy for obtaining hit matter</p>
</li>
<li>
<p>Establishment of a screening cascade</p>
</li>
<li>
<p>Defining appropriate pharmaceutical properties</p>
</li>
</ul>
<p>​      - Drug candidate selection</p>
<ul>
<li>Establishing a Target Product Profile</li>
</ul>
<h4 id="target-selection">Target selection</h4>
<ul>
<li>derived from published data that implicate</li>
</ul>
<p>​    - particular mutation</p>
<p>​    - RNA species</p>
<p>​    - protein as a driver of cancer</p>
<ul>
<li>
<p>always from clinical observations(bedside to bench)</p>
</li>
<li>
<p>derived from high throughput functional genome screens using</p>
</li>
</ul>
<p>​    - Si/ShRNA or</p>
<p>​    - CRISPR/cas9 screens</p>
<ul>
<li>define targets whose loss of function causes cancer cell death</li>
</ul>
<h4 id="target-validation">Target validation</h4>
<p>Biological assessment</p>
<ul>
<li>50−80% publications fail replication</li>
</ul>
<p>​    - lack of robustness in experimental design</p>
<p>​    - based on univariate statistical analysis to justify target selection from clinical data</p>
<p>​    - correlation not equal to causality</p>
<p>​    - identified genetic mutations may not represent a gain or loss of function</p>
<ul>
<li>prospective studies should be utilised</li>
</ul>
<p>Target validation</p>
<ul>
<li>
<p>often assessed by overexpression studies or knockdown of the target by Si/ShRNA or CRISPR/Cas 9 technologies</p>
</li>
<li>
<p>Over expression studies are often confounded by the supraphysiological levels of gene expression from an exogenous viral promoter, require careful titration</p>
</li>
<li>
<p>knockdown studies, off-target effects and inability to detect acute consequence of target knockdown</p>
</li>
</ul>
<p>​    - use at least three siRNA or guide RNA sequences at low nM concentrations</p>
<p>​    - make sure off-target effects are controlled for</p>
<p>​    - achieve a knockdown of 80%</p>
<p>​    - rescue any phenotype with SiRNA/gDNA resistant mutants</p>
<ul>
<li>use of antibiotic selection processes to select for cell clones where the target has been knocked out for genomic target validation</li>
</ul>
<p>​    -  up to 2 weeks</p>
<p>​    -  protein degradation-based technologies (dTAG system) can evaluate acute loss</p>
<ul>
<li>
<p>Chemical probes can used for target validation</p>
<p>- molecule modulators of target function should ideally be potent and selective for a protein target</p>
<p>- use of chemical probes in different cell contexts can reveal new biology</p>
<p>- resources: <a href="https://www.chemicalprobes.org/">Chemical Probes Portal</a> and  <a href="https://probeminer.icr.ac.uk/">Probe Miner</a></p>
</li>
<li>
<p>understand the target distribution in normal tissue, its homology to other proteins, enable an understanding of the potential therapeutic window and potential side effects</p>
</li>
</ul>
<p>Technical assessment</p>
<ul>
<li>
<p>establish the ability to drug a specific target protein</p>
<p>- directly via its active catalytic site or</p>
<p>- at a druggable cavity at a more distant site</p>
</li>
<li>
<p>Druggability: presence of protein structures within a target that enable interaction with chemical compounds</p>
</li>
<li>
<p>pharmaceutical drug-like properties:  Lipinski’s Rule of 5</p>
<p>- molecular weight of ≤500 Da</p>
<p>- Log P ≤5</p>
<p>- H-bond acceptors ≤10</p>
<p>- H donors ≤5</p>
</li>
<li>
<p>additional approaches</p>
<p>- precedence-based assessment</p>
<p>- knowledge of endogenous ligands for the target</p>
<p>- algorithms that predict druggable pockets based on structures found in the protein databases</p>
<p>- druggability based on machine learning approaches</p>
</li>
</ul>
<h4 id="establishing-a-screening-cascade">Establishing a Screening Cascade</h4>
<h5 id="hit-generation-strategy">Hit generation strategy</h5>
<ul>
<li>
<p>phenotypic screens</p>
<p>- focus on disease models and biological readouts</p>
<p>- not require knowledge of the target or pathways</p>
<p>- dependent on a predefined biological (in cell models) or physiological (in in vivo models) response</p>
<p>- suit for degrading target proteins rather than modulating their function</p>
</li>
<li>
<p>Combinatorial approach</p>
<p>- large chemical libraries WITH</p>
<p>- high throughput target based screening</p>
</li>
<li>
<p>For kinases and receptors, bio-chemical approaches have been used</p>
</li>
<li>
<p>For nuclear receptors, ion channels and membrane transcription factor cell-based assays are preferable</p>
</li>
</ul>
<h5 id="screening-methodologies-and-assays-for-the-cascade">Screening methodologies and assays for the cascade</h5>
<p>stablish a robust test cascade</p>
<p>- high throughput screening reactions in sessile 2D droplet micro-arrays</p>
<p>- Cell-based assays can also be used as phenotypic screens, based on function</p>
<p>- cells grown in 3D respond differently to treatment compared with cells in 2D cultures</p>
<p>- spheroids generates models</p>
<p>- mass spectrometry and nuclear magnetic resonance (NMR)</p>
<h4 id="choice-of-chemistry-approach">Choice of chemistry approach</h4>
<h4 id="generation-of-lead-compounds">Generation of Lead compounds</h4>
<p>- establish IC50 or GI50 from cellular/biophysical assay</p>
<p>- understand relationship between compound potency, target engagement, measured target modulation and the resulting phenotypic effect</p>
<h4 id="finding-the-target-engagement-biomarker-the-earlier-the-better">Finding the target engagement-biomarker-the earlier the better</h4>
<ul>
<li>
<p>enzymes with a known endogenous substrate, evaluation of target modulation</p>
</li>
<li>
<p>Mitotic targets are especially challenging</p>
<p>-  proliferation biomarker (P-HH3) and engineered cell lines can be used for in vivo but target is overexpressed</p>
</li>
<li>
<p>novel biology may not identified the biomarker of target engagement</p>
<p>- Tool compounds together with SiRNA or CRISPR may discover novel biomarkers</p>
<p>- obtained from gene expression profiling, proteomics, protein arrays, or metabolomics or a combination of these methods</p>
</li>
</ul>
<h4 id="secondary-pharmacology-profiling">Secondary pharmacology profiling</h4>
<p>- refer to ICH S7A, evaluate highlight potential off-target effects</p>
<p>- in vitro: hERG channel, CEREP screen</p>
<h4 id="further-in-vitro-testing-for-patient-selection">Further in vitro testing for patient selection</h4>
<p>required when</p>
<p>- original therapeutic hypothesis fails validation in preclinical models</p>
<p>- additional sensitive patient population are sought ahead of clinical studies</p>
<p>- provide information on potential mechanism of resistance or inform potential drug combinations</p>
<p>Organoids derived from patient material are more representative of patient tumors than spheroids</p>
<h4 id="optimising-the-pk-and-predicting-the-active-dose-in-man">Optimising the PK and predicting the active dose in man</h4>
<p>Understanding the target requirement from in vitro tests is a guide to optimising PK to demonstrate target engagement and efficacy</p>
<p><img src="https://i.loli.net/2021/03/08/HiM91TKd7EyulGg.png" alt="Figure3.1"></p>
<ul>
<li>
<p>compounds metabolised by aldehyde oxidase or carboxyles-terases are more challenging</p>
</li>
<li>
<p>Transport-mediated clearance is less accurate and always suffer drug resistance and efflux from many tumors and cancer stem cells</p>
</li>
<li>
<p>humanised mice models help to quantify metabolism</p>
</li>
<li>
<p>Identification of metabolites</p>
<p>- check of the potential activity of metabolites</p>
<p>- confirmation that no metabolites formed are toxic</p>
<p>- qualitative and quantitative evaluation of the metabolites formed across species favours selection of the toxicology species where metabolites are closest to those in man</p>
</li>
<li>
<p>more than 10% of parent compound in blood need to be measured in FiH</p>
</li>
<li>
<p>DDI: <a href="https://www.ema.europa.eu/en/documents/scientific-guide-line/guideline-investigation-drug-interactions/">EMA</a> and <a href="https://www.fda.gov/drugs/drug-interactions-labeling/drug-development-and-drug-interactions">FDA</a></p>
</li>
<li>
<p>PBPK and allometric scaling or variants  can be used to predict various PK parameters</p>
</li>
</ul>
<h4 id="selecting-the-optimal-in-vivo-model">Selecting the optimal in-vivo model</h4>
<ul>
<li>
<p>Xenograft models in immunocompromised mice (SCID, NCR nude, NOD)</p>
<p>- frequently utilised for PKPD and efficacy studies of non-immune- oncology targets</p>
<p>- lack of translation to clinical responses</p>
</li>
<li>
<p>patient derived xenografts (PDX)</p>
<p>- lower throughput</p>
<p>- better predict than simple xenografts derived from cell lines</p>
</li>
<li>
<p>genetically engineered mouse models</p>
<p>- validate the effect on the target and the resulting in vivo effect</p>
<p>- useful models in immuno-oncology</p>
</li>
<li>
<p>syngeneic mice provided the immune pathway targeted is similar in mice and humans</p>
<p>- suit for immuno-oncology target</p>
<p>- mimic the human immune response in mice</p>
<p>- still devoid of cytokines and growth factors</p>
<p>- successed in CAR-T therapy, NK therapy and PD-1 antibodies</p>
</li>
<li>
<p>PD assay developed originally in human cells may not be directly applicable in human tumor xenografts, PDX or GEMM models</p>
<p>- interactions of secondary antibody</p>
<p>- number of variants of a protein and their structure and regulation can vary across specie</p>
</li>
</ul>
<h3 id="drug-candidate-selection">Drug Candidate Selection</h3>
<p>Selection criteria for a drug development candidate to enable IND studies</p>
<p><img src="https://i.loli.net/2021/03/08/gX2i8WGStrluzjZ.png" alt="Table 3.2"></p>
<h3 id="translating-the-candidate-for-ind-and-fih">Translating the candidate for IND and FiH</h3>
<p>Once the candidate has been selected, a number of parallel yet interdependent activities need to be carried out.</p>
<p><strong>Toxicology evaluation</strong></p>
<p>- Help to define the starting dose in man and formulation</p>
<p>- In one rodent and one non-rodent species</p>
<p>- species selected relates to the target biology in the species</p>
<p>- knowledge from previous preclinical studies (ICH guidelines S7A and B, FDA Guidance for industry M3R2)</p>
<p>​    - antiproliferative activity often cause gut toxicity and induce myelosuppression</p>
<p>​    - Dogs are extremely prone to emesis, so mini-pigs can be a good alternative to dogs</p>
<p>- For immune-oncology and biologic agents, humanised mice models are often used alongside cynomolgus monkeys or minipigs</p>
<p>- For small molecules, it is important to consider the mechanism of clearance and the metabolites produced in the species tested, as well as impurities</p>
<p>- Duration of the study should cover all potential clinical applications</p>
<p>- ICH guidelines S7A and B, <a href="https://www.fda.gov/animal-veterinary/guidance-regulations/guidance-industry">FDA Guidance for industry M3R2</a></p>
<p>- In oncology, a 28-day evaluation is usually sufficient for IND application</p>
<p><strong>Definition of dose and scheduling for FIH</strong></p>
<p>starting dose</p>
<p>For cytotoxic agents:</p>
<p>- NOAEL dose multiplied by safety margin</p>
<p>- combined with in vivo PK/PD and efficacy models to</p>
<p>- define systemic exposure</p>
<p>- correlate with the degree of target engagement and desired efficacy</p>
<p>- use allometric scaling to extrapolate doses from animal to human based on BSA(ICH guidelines S7A, S7B, FDA Guidelines for industry M3R2)</p>
<p><strong>Formulation</strong></p>
<p>- For IV, solution stability needed</p>
<p>- For PO, rate of dissolution, crystalline form, excipients are important</p>
<p>- low solubility of the salt form has resulted in variable bioavailability</p>
<p><strong>Clinically validated biomarker assays</strong></p>
<p>assays to measure biomarkers of target engagement is essential</p>
<p>- In solid tumors, the biomarker selected to support preclinical studies is ideally validated for human tumors</p>
<p>- limited by the availability of tumor biopsies</p>
<p>- paired biopsies: before treatment and at peak effect</p>
<p>- liquid biopsies: circulating free DNA</p>
<p>- surrogate tissues(blood cells, hair follicles, skin biopsies, plasma) inform to generate concentration-time-target engagement relationship</p>
<p>- In haematological malignancies</p>
<p>- circulating tumors are available</p>
<p>- often limited by the myelosuppressive effect of therapeutics</p>
<p>- biomarker modulations techniques</p>
<p>- immunoassays in a variety of formats and platforms</p>
<p>- gene expression profiling</p>
<p>- flow cytometry</p>
<p>- proteomics and metabolomics</p>
<p>- target established drivers of cancer in patient subpopulations, excluding patients with certain molecular features</p>
<h4 id="modelling-and-simulations">Modelling and Simulations</h4>
<p><strong>Cases</strong></p>
<ul>
<li>
<p>ALK: growth inhibition is observed when EC50 to EC60 of ALK is reached</p>
</li>
<li>
<p>PI3K: a minimum of 30% inhibition of AKT phosphorylation is required for antitumor activity in breast cancer</p>
</li>
<li>
<p>gefitinib and sorafenib: assuming that free steady state concentrations in human and mouse were identical, predicted correct clinical doses</p>
</li>
<li>
<p>resistance model: predict the active doses for sensitive and resistant patients</p>
</li>
<li>
<p>immune oncology: tumor uptake for cytokine-based immunotherapy model available</p>
</li>
<li>
<p>Toxicities can be modelled and to optimise scheduling</p>
</li>
</ul>
<h4 id="development-of-a-target-product-profile-tpp">Development of a Target Product Profile (TPP)</h4>
<p>highly useful for IND &amp; as an important dynamic document, at the pre-IND stage it should include</p>
<p>- potential indications and usage in defined patient populations</p>
<p>- dosage and administration</p>
<p>- formulation (with dosage forms and strengths)</p>
<p>- contraindications</p>
<p>- clinical pharmacology</p>
<p>- non-clinical toxicology</p>
<p>- planned clinical studies</p>
<p>- storage, handling and drug stability</p>
<p>- patient information and the consent form for the first in human study</p>


  <footer>
  
  

  
  <hr>
  <div class="copyright">© <a href="/">illiterate</a> 2021 - 2025</div>
  
  </footer>
  </article>
  
  </body>
</html>

